multiqc/utils/search_patterns.yaml

```diff
@@ -429,6 +429,10 @@ homer/FreqDistribution:
   fn: "petag.FreqDistribution_1000.txt"
 hops:
   fn: "heatmap_overview_Wevid.json"
+hostile:
+  fn: "*.json"
+  contents: '"reads_in"'
```

Comment:
Can the pattern be more specific than `"reads_in"`? Maybe `"reads_removed_proportion"` would be more unique. It's unfortunate there is no `"tool": "hostile"` in the JSON that would allow to detect it unambigously :(